privacy/scrubber_test.go

```diff
@@ -38,12 +39,12 @@ func TestScrubDevice(t *testing.T) {
 				MACMD5:   "",
 				IFA:      "",
 				IP:       "1.2.3.0",
-				IPv6:     "2001:0db8:0000:0000:0000:ff00:0:0",
+				IPv6:     "2001:db8:2233:4400::",
```

Comment:
In this case `0db8` and `db8` are identical. Because we use `net.ParseIP(ip)` and then `ipMasked.String()` returns it in this form.

Reply:
https://www.ciscopress.com/articles/article.asp?p=2803866 -> paragraph "Rule 1: Omit Leading 0s"

Reply:
Gotcha. Maybe just add a the `0` to keep this straight forward?

Reply:
Ooof, I see. This value is returned by the go `net.IP` functionality. In order to make this change we will need to compare net.IPs, not strings. This will require some assertions refactoring. We now assert the whole `expected: &openrtb2.Device` with the result.

We will need to change this(current implementation): 
```go
assert.Equal(t, test.expected, result, test.description)
```
to something like this:
```go
assert.Equal(t, net.ParseIP(test.expected.IP).String(), net.ParseIP(result.IP).String(), test.description)
assert.Equal(t, net.ParseIP(test.expected.IPv6).String(), net.ParseIP(result.IPv6).String(), test.description)
assert.Equal(t, test.expected.DIDMD5, result.DIDMD5, test.description)
assert.Equal(t, test.expected.IFA, result.IFA, test.description)
...
```

 Are we good to go with this format?

Reply:
How about we just change 0 to 1? My only intention here was to keep the test straight forward and not confuse the reader with the additional dimension of leading 0 optimization.

Reply:
How about we change to 0 to a 1 to remove this issue entirely?